Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/nexus/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared', 'iqAzureDevops-pipeline-library']) _
+
+dockerizedBuildPipeline(
+  prepare: {
+    githubStatusUpdate('pending')
+  },
+
+  buildAndTest: {
+    sh './build.sh'
+  },
+
+  skipVulnerabilityScan: true,
+
+  vulnerabilityScan: {
```

Comment:
if `skipVulnerabilityScan` is true, can this block be removed?

Reply:
removed.